code-of-conduct/COC_PyBay.md

```diff
@@ -0,0 +1,75 @@
+ - **Be respectful.** Not all of us will agree all the time, but disagreement is no excuse for poor behavior and poor manners. We might all experience some frustration now and then, but we cannot allow that frustration to turn into a personal attack. It’s important to remember that a community where people feel uncomfortable or threatened is not a productive one. Members of the Python community should be respectful when dealing with other members as well as with people outside the community.
+ - **Be careful in the words that you choose.** We are a community of professionals, and we conduct ourselves professionally. Be kind to others. Do not insult or put down other participants, individually or as a group. Harassment and other exclusionary behavior aren't acceptable. This includes, but is not limited to:
+
+     - Violent threats or language directed against another person or yourself.
+     - Discriminatory jokes and language.
+     - Posting sexually explicit or violent material.
+     - Posting (or threatening to post) other people's personally identifying information ("doxing").
+     - Personal insults, especially those using racist or sexist terms.
+     - Unwelcome sexual attention.
+     - Deliberate intimidation, stalking, or following.
+     - Harassing photography or recording, or photographing or recording people who have explicitly declined to be recorded.
+     - Inappropriate physical contact.
+     - Advocating for, or encouraging, any of the above behavior.
+     - Repeated harassment of others. In general, if someone asks you to stop, then stop.
+
+##Where does the Code of Conduct apply
+
+This Code of Conduct applies to all spaces managed by PyBay. This includes:
+
+ - Conferences (including social events and peripheral activities, both virtual and in-person)
+ - Unconferences and sprints
+ - Meetups
+ - Workshops
+ - Presentation materials used in talks or sessions
+ - Slack
+ - Mailing lists
+ - GitHub
+ - Social media hashtags
+ - meetup.com discussion boards
+ - Any other forums created by the community which the community uses for communication.
+
+In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+Speakers, presenters, and open space organizers
+
+Speakers, presenters, and those organizing open spaces or similar events are expected to uphold the community guidelines. Please let a member of staff know immediately if there are any issues during one of these events. Presentations, open spaces, and similar events will generally not be stopped for one-time gaffes or minor problems, although a staff member will speak to the presenter afterward and we will include any incidents in our transparency report. However, staff will take immediate action to politely and calmly stop any presentation or event that repeatedly or seriously violates our Code of Conduct policy.
```

Comment:
###Speakers, presenters, and open space organizers

Reply:
changed